Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -9505,7 +9505,7 @@
 											"    pm.test(\"Every non-primary key value on the entity should be changed\",\r",
 											"    () => {\r",
 											"        // Traverse all properties that are not pripary keys\r",
-											"        for (const [key, value] of Object.entries(originalEntity).filter(function (e) { return this.indexOf(e[0]) < 0; }, keys)) {\r",
+											"        for (const key of Object.keys(originalEntity).filter(function (e) { return this.indexOf(e) < 0; }, keys)) {\r",
```

Comment:
both Get requests in Profile definitions can include and exclude certain properties on resources / Include are still using Object.entries